File: baekjoon/js/level4/2562.js
const fs = require('fs')
const filePath = process.platform === 'linux' ? '/dev/stdin' : '../input.txt';
const input = fs.readFileSync(filePath).toString().trim().split(/\s/).map(Number);

const max = Math.max(...input);
const idx = input.findIndex((elem) => elem === max) + 1;

console.log(max + '\n' + idx);

File: baekjoon/cpp/level5/2562.c
#include<stdio.h>

int main(){
    int arr[9];

    for(int i=0; i<9; i++)
        scanf("%d",arr+i);
    
    int max = *arr, idx=0;

    for(int i=1; i<9; i++){
        if(max < *(arr+i)){
            max = *(arr+i);
            idx = i;
        }
    }

    printf("%d\n%d",max,idx+1);
}